npe2/manifest/contributions/_menus.py

```diff
@@ -1,27 +1,26 @@
+from collections import namedtuple
 from typing import Optional, Union
 
 from pydantic import BaseModel, Field
 
 from ..utils import Executable
 
-# # napari provides these
-# class Menu(BaseModel):
-#     key: str
-#     id: int
-#     description: str
-#     supports_submenus: bool = True
-#     deprecation_message: Optional[str]
-
-
-# napari_menus = [
-#     Menu(key="command_pallete", id=0, description="The Command Palette"),
-#     Menu(
-#         key="layers__context", id=1, description="The layer list context menu"
-#     ),
-#     Menu(
-#         key="layers__context", id=1, description="The layer list context menu"
-#     ),
-# ]
+Menu = namedtuple("Menu", "key description")
+
+# a list of valid napari menu locations that plugins can contribute too.
+# keys provided in the plugin manifest must match these strings
+napari_menus = [
```

Comment:
what about using a dict instead of a nx2 array? this would make it easier to get just the paths since u could go `napari_menus.keys()`